.bloodhound.yml

```diff
@@ -5,6 +5,7 @@ k8s_versions:
   - 1.29.9
   - 1.30.5
   - 1.31.1
+  - 1.32.0
```

Comment:
Could you remove 1.28.x and 1.29.x versions?

Reply:
Sure will do that